src/aiperf/cli_commands/speed_bench_report.py

```
@@ -0,0 +1,49 @@
+from typing import Literal
+
+from cyclopts import App
+
+app = App(name="speed-bench-report")
+
+
+@app.default
+def speed_bench_report(
+    paths: list[Path],
+    output: Path = Path("speed_bench_report.csv"),
+    format: Literal["csv", "table", "both"] = "both",
```

Ruff (0.15.10)

[error] Function argument `format` is shadowing a Python builtin

(A002)